### ISSUE TITLE
Responsive Issues in CSS Grid

### ISSUE THREAD

**Issue by ZachHaber** [emeeks/semiotic#317]
There are two issues I've noticed with responsive frames in a css grid.
1. The responsive height doesn't work.
2. The responsive width doesn't work if there is a unit of 'fr' or fractional unit.
In both cases, the way it acts is as follows: 
It will respond to increases in size and increase its size to match. It will, however, not respond to decreases in size.
Here's a link to a codepen fork that has the behavior. You can change the grid-template-columns to 100% instead of 1fr and you'll notice that responsiveWidth suddenly starts working normally.
https://codepen.io/ZachHaber/pen/zJNyWZ
Once again, thanks for your work in maintaining this library! 😄

**Comment by emeeks** [comment]
This is interesting and I'm not sure how to fix it. Basically grid is resizing because the Semiotic frames are set to a size smaller than it but it's not resizing smaller because it's already filled with sized elements. This is going to take some research.

**Comment by ZachHaber** [comment]
So, thanks to your comment, and a quick google search, I had an idea for a workaround: Wrap the frame in a div with min-width/min-height =0. It's part of how the grid works. Sadly, that doesn't work on its own when added directly to the responsive-container wrapper component.
https://codepen.io/ZachHaber/pen/OoggLJ
Let me know if there's anyway I can help figure out a full fix for the issue rather than a workaround.

**Comment by emeeks** [comment]
Ah, you're right. In which case it looks like if you just add the following to your CSS it works fine:

```
.responsive-container {
      min-width: 1px;
    min-height: 1px;
}
```

**Comment by ZachHaber** [comment]
I just tried that CSS selector on the codepen I put up earlier, and it doesn't seem to work for me without the wrapper div around the responsive frame.

**Comment by emeeks** [comment]
Ah, you're right, I hadn't realized the pen had that wrapper in it. I'd rather not ship with so many nested divs, let me see if I can find a better way. If you find one, feel free to file a PR.

**Comment by emeeks** [comment]
As a stopgap I added a `gridDisplay` prop that will adjust the style of the responsive wrapper div that seems to give it settings that work with `display: grid`. It's available in 1.14.3
